### DISCUSSION TITLE
AppId google store

### DISCUSSION THREAD

**Discussion by HectorDev15** [quasarframework/quasar#10493] [CLI - Capacitor / Cordova modes]
Hi Devs i will upload an app in store but i need set appid, after created project where i can configure that?.

**Comment by pikil** [comment]
For cordova, that is quasar.conf file:
```
cordova: {
      id: 'org.cordova.your.app'
      ....
}
```
For capacitor, that's capacitor.config.json:
```
{
  "appId": "org.capacitor.your.app",
  ....
}
```

_* I've used different appIds in the examples above, but that should be a single appId for one project, just set it the same for any mode you use, whether it is Cordova or Capacitor._

_** Quasar might have other, more convenient places to fetch appIds from, so if someone knows better ones, would be great to see the answer here as well :)_

**Comment by ontwikkelfabriek** [comment]
@HectorDev15 


And make sure that the id you choose is **valid** for the Google Play **and** the Itunes store.
